spec/requests/api/v2/allocations_spec.rb

```diff
@@ -50,6 +72,10 @@ def when_invalid_parameters_are_posted_to_the_allocations_endpoint
     post "/api/v2/providers/#{@accredited_body.provider_code}/allocations", params: { allocation: { provider_id: invalid_provider_id } }, headers: { "HTTP_AUTHORIZATION" => @credentials }
   end
 
+  def when_i_get_the_allocations_index_endpoint
+    get "/api/v2/providers/#{@accredited_body.provider_code}/allocations", headers: { "HTTP_AUTHORIZATION" => @credentials }
```

Comment:
Just wondering why not use the railsy path?

Reply:
I think if you hard code the URL then you're testing the routes are resolving as you expect. If you use the path helpers then you could have a mistake in the route and the test still pass. So, I'd go with hard coding it at this top level of testing.